### PR TITLE
Adds abandoned crates to lavaland (sort of)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
When a tendril dies, it will drop a chest. You have 5 seconds to grab the chest and get clear before the area collapses in on itself as the tendril dies. The chest contains random spooky themed junk.

Spooky chest sprites are from my girlfriend.

:cl: Kor
rscadd: Killing a necropolis tendril will now drop a chest full of spooky loot.
rscadd: Killing a necropolis tendril will now cause the ground to collapse around it after 5 seconds.
/:cl:

### CODE REVIEW

**Comment by Zilenan91** [comment]
I'm glad it's not just a crate full of ties like the old abandoned crates

File: code/modules/mining/abandoned_crates.dm [hunk 1]
```diff
@@ -214,4 +214,76 @@
...
+			new /obj/item/clothing/head/culthood(src)
+			new /obj/item/clothing/suit/cultrobes(src)
+		if(13)
+			new /obj/item/weapon/sord(src)
+		if(14)
+			new /obj/item/weapon/nullrod/claymore/darkblade
+		if(15)
+			new /obj/item/weapon/nullrod/armblade(src)
+		if(16)
+			new /obj/item/weapon/guardiancreator(src)
```
Comment (ChangelingRain) [inline code/modules/mining/abandoned_crates.dm]: Make it a holoparasite injector.

Comment (KorPhaeron) [inline code/modules/mining/abandoned_crates.dm]: That aint spooky

File: code/modules/mining/abandoned_crates.dm [hunk 2]
```diff
@@ -214,4 +214,76 @@
...
+	name = "necropolis chest"
+	desc = "It's watching you closely."
+	icon_state = "necrocrate"
+
+/obj/structure/closet/crate/necropolis/tendril
+	desc = "It's watching you suspiciously."
+
+/obj/structure/closet/crate/necropolis/tendril/New()
+	..()
+	var/loot = rand(1,25) //100 different crates with varying chances of spawning
```
Comment (Supermichael777) [inline code/modules/mining/abandoned_crates.dm]: assuming this will get filled out later

Comment (RemieRichards) [inline code/modules/mining/abandoned_crates.dm]: nah he just copypasted it from abandoned crates, which really do have 100.

**Comment by Supermichael777** [comment]
I feel lava-lands enviro hazards are a bit too lethal to be creating them on the fly. Also creating permanent hazards FROM the destruction of permanent hazards is bad is bad. Please implement ways to mitigate chasms before making more ways to make them.  Real extendable bridges when?

File: code/modules/mining/abandoned_crates.dm [hunk 3]
```diff
@@ -214,4 +214,76 @@
...
+		if(15)
+			new /obj/item/weapon/nullrod/armblade(src)
+		if(16)
+			new /obj/item/weapon/guardiancreator(src)
+		if(17)
+			new /obj/item/stack/sheet/runed_metal/fifty(src)
+		if(18)
+			new /obj/item/weapon/kitchen/knife/ritual(src)
+		if(19)
+			new /obj/item/weapon/coin/antagtoken(src)
```
Comment (Supermichael777) [inline code/modules/mining/abandoned_crates.dm]: seems kind of week comparitivly

File: code/modules/mining/mine_turfs.dm [hunk 4]
```diff
@@ -692,10 +692,14 @@ var/global/list/rockTurfEdgeCache
 	temperature = 300
 	baseturf = /turf/simulated/chasm/straight_down/lava_land_surface
 
+/turf/simulated/chasm/straight_down/lava_land_surface/drop(atom/movable/AM)
+	visible_message("[AM] falls into [src]!")
+	qdel(AM)
```
Comment (KazeEspada) [inline code/modules/mining/mine_turfs.dm]: BINARY DEATH YAY

Comment (KorPhaeron) [inline code/modules/mining/mine_turfs.dm]: I'l let you know when I have time to code another Z level.

Also the death isn't a binary "you have a suit/don't have a suit," it's a choice to go for the loot or run to safety, factoring in how close you are (did you kill it with a gun? Or get up close), how many mobs are about/how they might impede you, how fast you can still run, etc.

**Pull request merged**